### ISSUE TITLE
[regression] service fails to correctly determine whether a service is already running on non-systemd systems in 2.2.1rc3

### ISSUE THREAD

**Issue by mgedmin** [ansible/ansible#20084]
##### ISSUE TYPE
 - Bug Report

##### COMPONENT NAME
`service` module

##### ANSIBLE VERSION
```
ansible 2.2.1.0 (detached HEAD d8c9b8d347) last updated 2017/01/10 13:40:37 (GMT +300)
  lib/ansible/modules/core: (detached HEAD a9d844c089) last updated 2017/01/10 13:40:47 (GMT +300)
  lib/ansible/modules/extras: (detached HEAD 19d20ca1a0) last updated 2017/01/10 13:40:49 (GMT +300)
  config file = /home/mg/src/deployments/provisioning/ansible.cfg
  configured module search path = Default w/o overrides
```

##### OS / ENVIRONMENT
Target: Ubuntu 14.04 LTS

##### SUMMARY
The `service` module reports `changed` for services that are already running.

This is a regression since 2.2.0 final.

This bug is not present in current `devel` (git bcf9cd1).

This bug affects remote targets running Ubuntu 14.04 LTS but not Ubuntu 16.04 LTS.

I haven't actually verified this, but from what I can tell in the code there's a more serious consequence: due to the inverted check `service: state=started` will fail to start services that are not running, and `service: state=stopped` will fail to stop services that are running.

##### STEPS TO REPRODUCE

```yaml
- hosts: all
  gather_facts: no
  tasks:
    - service: name=postfix state=started
```

- `ansible-playbook -i example.com, -u root test.yml -C`

The same thing happens if you omit `-C`.

##### EXPECTED RESULTS

```
PLAY [all] *********************************************************************

TASK [service] *****************************************************************
ok: [iv-1.pov.lt]

TASK [assert] ******************************************************************
ok: [iv-1.pov.lt] => {
    "changed": false, 
    "msg": "All assertions passed"
}

PLAY RECAP *********************************************************************
iv-1.pov.lt                : ok=2    changed=0    unreachable=0    failed=0   
```

##### ACTUAL RESULTS

```
ansible-playbook -i iv-1.pov.lt, -u root -CD test.yml

PLAY [all] *********************************************************************

TASK [service] *****************************************************************
changed: [iv-1.pov.lt]

TASK [assert] ******************************************************************
fatal: [iv-1.pov.lt]: FAILED! => {
    "assertion": "not foo.changed", 
    "changed": false, 
    "evaluated_to": false, 
    "failed": true
}
	to retry, use: --limit @/home/mg/src/bugs/ansible-bug/test.retry

PLAY RECAP *********************************************************************
iv-1.pov.lt                : ok=1    changed=1    unreachable=0    failed=1   
```

**Comment by mgedmin** [comment]
git bisect blames commit d8c9b8d347db9032b4d7486a21c1719c92e2c130 which basically did nothing except update submodules.

**Comment by mgedmin** [comment]
git bisect in `lib/modules/core` blames https://github.com/ansible/ansible-modules-core/commit/6e558aa4296b00d1056eb773bb0d6ab81b1a0f11

**Comment by bcoca** [comment]
fixed via afbe3bd63f9e347d418643953e8a1c82f625877d and backported
